pkg/catalog/interrogate.go

```diff
@@ -172,15 +172,13 @@ func ImageForRole(
 			break
```

Comment:
At this point, we don't even need to declare repoTag outside the loop. We could just return immediately once we find the role, since the role-specific image will always be defined now. And the comment above the loop can be changed to something like "// Find the image for the role." or just deleted altogether.

Interestingly, if we do manage to get all the way through the loop without finding the role, something is wrong. In that case we should `return "", err`